src/java.base/share/classes/java/lang/classfile/CodeBuilder.java

```diff
@@ -545,48 +493,19 @@ default CodeBuilder invokeInstruction(Opcode opcode, MemberRefEntry ref) {
      * @param isInterface the interface method invocation indication
      * @return this builder
      */
-    default CodeBuilder invokeInstruction(Opcode opcode, ClassDesc owner, String name, MethodTypeDesc desc, boolean isInterface) {
-        return invokeInstruction(opcode,
+    default CodeBuilder invoke(Opcode opcode, ClassDesc owner, String name, MethodTypeDesc desc, boolean isInterface) {
+        return invoke(opcode,
                 isInterface ? constantPool().interfaceMethodRefEntry(owner, name, desc)
                             : constantPool().methodRefEntry(owner, name, desc));
     }
 
-    /**
-     * Generate an instruction to invoke a dynamically-computed call site
-     * @param ref the dynamic call site
-     * @return this builder
-     */
-    default CodeBuilder invokeDynamicInstruction(InvokeDynamicEntry ref) {
-        with(InvokeDynamicInstruction.of(ref));
-        return this;
-    }
-
-    /**
-     * Generate an instruction to invoke a dynamically-computed call site
-     * @param desc the dynamic call site
-     * @return this builder
-     */
-    default CodeBuilder invokeDynamicInstruction(DynamicCallSiteDesc desc) {
-        MethodHandleEntry bsMethod = handleDescToHandleInfo(constantPool(), (DirectMethodHandleDesc) desc.bootstrapMethod());
-        var cpArgs = desc.bootstrapArgs();
-        List<LoadableConstantEntry> bsArguments = new ArrayList<>(cpArgs.length);
-        for (var constantValue : cpArgs) {
-            bsArguments.add(BytecodeHelpers.constantEntry(constantPool(), constantValue));
-        }
-        BootstrapMethodEntry bm = constantPool().bsmEntry(bsMethod, bsArguments);
-        NameAndTypeEntry nameAndType = constantPool().nameAndTypeEntry(desc.invocationName(), desc.invocationType());
-        invokeDynamicInstruction(constantPool().invokeDynamicEntry(bm, nameAndType));
-        return this;
-    }
-
     /**
      * Generate an instruction to create a new object
      * @param type the object type
      * @return this builder
      */
-    default CodeBuilder newObjectInstruction(ClassEntry type) {
-        with(NewObjectInstruction.of(type));
-        return this;
+    default CodeBuilder newObject(ClassEntry type) {
```

Comment:
The two `newObject` methods seem to fit in the pattern of methods that are being removed, since they don't differentiate sufficiently with the `new_` methods that defer to them.

Reply:
Right, we can remove also the `newObject` and keep `new_` methods, thanks.